docs/running-headscale-container.md

```diff
@@ -124,7 +117,7 @@ To register a machine when running `headscale` in a container, take the headscal
 
 ```shell
 docker exec headscale \
-  headscale --user myfirstuser nodes register --key <YOU_+MACHINE_KEY>
+  headscale --user myfirstuser nodes register --key <YOUR_MACHINE_KEY>
```

Comment:
```suggestion
  --user myfirstuser nodes register --key <YOUR_MACHINE_KEY>
```

Reply:
For `exec` the command is actually `headscale ...`, I tested the commands with docker on Debian 12.